README.md

```diff
@@ -149,6 +175,11 @@ Key variables:
 | `INSECURE` | Skip certificate validation | true |
 | `COLLABORA_DOMAIN` | Collabora domain | collabora.opencloud.test |
 | `WOPISERVER_DOMAIN` | WOPI server domain | wopiserver.opencloud.test |
+| `KEYCLOAK_DOMAIN` | Keycloak domain | keycloak.opencloud.test |
+| `KEYCLOAK_ADMIN_USER` | Keycloak admin username | admin |
+| `KEYCLOAK_ADMIN_PASSWORD` | Keycloak admin password | admin |
+| `KEYCLOAK_REALM` | Keycloak realm name | openCloud |
```

Comment:
The KEYCLOAK_DOMAIN is only needed for dns routing. We should probably stick to the original env vars:


```suggestion
| `KC_HOSTNAME` | Keycloak hostname | keycloak.opencloud.test |
| `KEYCLOAK_ADMIN` | Keycloak admin username | admin |
| `KEYCLOAK_ADMIN_PASSWORD` | Keycloak admin password | admin |
| `OC_OIDC_ISSUER` | OpenID Connect Issuer | https://keycloak.opencloud.test/realms/openCloud |
```

Reply:
kind of refactored it, but i kept `KEYCLOAK_DOMAIN`